demisto_sdk/commands/lint/linter.py

```diff
@@ -275,16 +264,6 @@ def _run_lint_in_host(self, no_flake8: bool, no_bandit: bool, no_mypy: bool, no_
                 if exit_code:
                     self._pkg_lint_status["exit_code"] |= EXIT_CODES[lint_check]
                     self._pkg_lint_status[f"{lint_check}_errors"] = output
-        if self._facts['lint_unittest_files']:
-            for lint_check in ["flake8"]:
-                exit_code: int = SUCCESS
-                output: str = ""
-                if lint_check == "flake8" and not no_flake8:
-                    exit_code, output = self._run_flake8(py_num=self._facts["images"][0][1],
-                                                         lint_files=self._facts["lint_unittest_files"])
-                if exit_code:
-                    self._pkg_lint_status["exit_code"] |= EXIT_CODES[lint_check]
-                    self._pkg_lint_status[f"{lint_check}_errors"] = output
```

Comment:
Is this condition the cause of the duplication?

Reply:
Same as above